capreolus/evaluator.py

```diff
@@ -5,9 +5,11 @@
 
 from capreolus.searcher import Searcher
 from capreolus.utils.loginit import get_logger
+from capreolus.eval.msmarco_eval import compute_metrics_from_files
```

Comment:
I'm curious why we calculate MRR this way instead of using pytrec_eval with recip_rank?